docs/pages/eas/workflows/syntax.mdx

```diff
@@ -0,0 +1,440 @@
+    app_version: string # optional
+    git_commit_hash: string # optional
+    fingerprint_hash: string # optional
+    sdk_version: string # optional
+    runtime_version: string # optional
+    simulator: boolean # optional
+```
+
+#### `submit`
+
+Submits an Android or iOS build to the app store using [EAS Submit](/submit/introduction/).
+
+```yaml
+submit:
+  # @info #
+  type: submit
+  # @end #
+  environment: production | preview | development # optional
+  params:
+    build_id: string # required
+    profile: string # optional, default: production
+```
+
+#### `update`
+
+Publishes an update using [EAS Update](/eas-update/introduction/).
+
+```yaml
+update:
+  # @info #
+  type: update
+  # @end #
+  environment: production | preview | development # optional
+  params:
+    message: string # optional
+    platform: string # optional - android, ios, or all
+    branch: string # optional
+    channel: string # optional - cannot be used with branch
+```
+
+## Custom jobs
+
+Runs custom code and can use built-in EAS functions. Does not require a `type` field.
+
+```yaml
+custom_job:
+  runs_on: linux-medium | linux-large | macos-medium | macos-large | linux-medium-nested-virtualization | linux-large-nested-virtualization # optional, default: linux-medium
+  environment: production | preview | development # optional
+  steps:
+    # ...
+```
+
+### `jobs.<job_id>.steps`
+
+A job contains a sequence of tasks called `steps`. Steps can run commands or use an action.
+
+```yaml
+jobs:
+  custom_job:
+    # @info #
+    steps:
+      - name: My first step
+        run: echo "Hello World"
+    # @end #
+```
+
+### `jobs.<job_id>.outputs`
+
+A list of outputs for the job. Job outputs are available to all downstream jobs that depend on this job. Set outputs in a step using the `set-output` function.
+
+```yaml
+jobs:
+  job_1:
+    # @info #
+    outputs:
+      output_1: ${{ steps.step_1.outputs.test }}
+    # @end #
+    steps:
+      - id: step_1
+        run: set-output test "hello world"
+
+  job_2:
+    needs: [job_1]
+    steps:
+      # @info #
+      - run: echo ${{ needs.job_1.outputs.output_1 }}
+      # @end #
+```
+
+### Built-in EAS functions
+
+EAS provides a set of built-in reusable functions that you can use in a workflow without defining the function definition.
+
+#### `eas/checkout`
+
+Checks out your project source files.
+
+For example, a workflow with the following `steps` will check out the project and list the files in the **assets** directory:
+
+```yaml
+custom_job:
+  steps:
+    - uses: eas/checkout
+```
+
+<BoxLink
+  title="eas/checkout source code"
+  description="View the source code for the eas/checkout function on GitHub."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/checkout.ts"
+/>
+
+#### `eas/install_node_modules`
+
+Installs node modules using the package manager (npm, pnpm, or Yarn) detected based on your project. Works with monorepos.
```

Comment:
```suggestion
Installs node modules using the package manager (bun, npm, pnpm, or Yarn) detected based on your project. Works with monorepos.
```

Not sure if it is specific, but we do support bun so it is good to mention it here.